gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -2828,6 +2828,8 @@ public String getIconClass(final Assignment assignment) {
 		// "Lesson Builder" is currently hardcoded in SimplePageBean.java (no localization required)
 		} else if (StringUtils.equals(externalAppName, "Lesson Builder")) {
 			iconClass = getLessonBuilderIconClass();
+		} else if (StringUtils.equals(externalAppName, "Attendance")) {
```

Comment:
I prefer using a switch statement here but I'm fine with the PR, just change the necessary lines.